packages/nuxt/src/components/plugins/tree-shake.ts

```
@@ -197,62 +191,61 @@
-              } else if (node.type === 'MemberExpression') {
-                // dev only with $setup or _ctx
-                found = (node.property.type === 'Literal' && node.property.value === name) || (node.property.type === 'Identifier' && node.property.name === name)
-              }
-            },
-          })
-        }
-      },
-    })
-    if (!found) { return name }
-  }
+function isComponentNotCalledInSetup (code: string, id: string, name: string): string | void {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)